signal_timeout_block.py

```diff
@@ -33,7 +36,7 @@ class SignalTimeout(GroupBy, Block):
     """
 
     intervals = ListProperty(Interval, title='Timeout Intervals')
-    version = VersionProperty('1.0.0')
+    version = VersionProperty('1.1.0')
```

Comment:
It's definitely close to being a 2.0.0.  The block was documented as having an output signal with 'timeout' and 'group' attributes. Now it has those two attributes plus more. I suppose that is enough to potentially break backwards compatibility in some services..